packages/sitecore-jss/src/layout/rest-layout-service.test.ts

```diff
@@ -145,12 +130,14 @@ describe('RestLayoutService', () => {
 
     return service
       .fetchLayoutData('/home', 'da-DK', req, res)
-      .then((layoutServiceData: LayoutServiceData & AxiosRequestConfig) => {
-        expect(layoutServiceData.headers.cookie).to.equal('test-cookie-value');
-        expect(layoutServiceData.headers.referer).to.equal('http://sctest');
-        expect(layoutServiceData.headers['user-agent']).to.equal('test-user-agent-value');
-        expect(layoutServiceData.headers['X-Forwarded-For']).to.equal('192.168.1.10');
-        expect(layoutServiceData.data).to.deep.equal({
+      .then((layoutServiceData: LayoutServiceData & NativeDataFetcherConfig) => {
+        if (layoutServiceData.headers instanceof Headers) {
```

Comment:
`layoutServiceData`  can't contain `headers`, so you are not verifying headers anymore
Check rest of unit tests

Reply:
@addy-pathania we are still missing `headers` configuration and verification in unit tests